docs/decisions/0003-initial-api-specification.rst

```diff
@@ -0,0 +1,79 @@
+0003 Initial API Specification
+##############################
+
+Status
+******
+
+**Provisional**
+
+Context
+*******
+
+This doc describes the API specification for the ``subsidy-access-policy`` Django app in this repo.
+
+
+
+Open Questions and TODOs
+========================
+
+
+
+Primary Access Policy Route
+***************************
+**/api/v1/policy/**
+The root URL for all actions allowed on access policy apis.
+
+
+Enterprise Access Policy Transactions
+**************************************
+
+POST access policy redeem transaction
+======================================
+**/api/v1/policy/redeem/**
+
+Creats a new transaction to redeem an entitlement by making a request to ``POST /api/v1/subsidies/.../transactions/``
+A side-effect of a successful POST request here is the creation of a course enrollment or entitlement
+that "fulfills" the ledger transaction.
+
+Inputs
+------
+
+- ``customer-id`` (POST data, required): The uuid of the customer. For now it will be an enterprise customer uuid.
+- ``learner_id`` (POST data, required): The user for whom the transaction is written and for which a enrollment should occur.
+- ``content_key`` (POST data, required): The content for which a enrollment is created.
+
+Outputs
+-------
+Returns data about the transaction.
+
+::
+
+   {
+       'uuid': 'the-transaction-uuid',
+       'status': 'completed',
+       'idempotency_key': 'the-idempotency-key',
```

Comment:
I think the transaction uuid will be unique and can be used for duplicate check

Reply:
The response is not finalized.